### PR TITLE
Mccalluc/use data access level

### LINKED ISSUE #970: fix access group vs access level in facets
1. `data_access_level` should be added to facets as "Data Access".
1. `access_group` facet should be removed.

### PR DESCRIPTION
**Pull request by mccalluc**
Fix #970.

### CODE REVIEW

File: context/app/static/js/components/Detail/SummaryData/SummaryData.jsx [hunk 1]
```diff
@@ -17,7 +16,7 @@ function SummaryData(props) {
           <FlexCenterAlign>
             <StatusIcon status={status} />
             <SummaryItem>{status}</SummaryItem>
-            <SummaryItem>{`${capitalizeString(data_access_level)} Access`}</SummaryItem>
+            <SummaryItem>{`${mapped_data_access_level} Access`}</SummaryItem>
```
Comment (john-conroy) [inline context/app/static/js/components/Detail/SummaryData/SummaryData.jsx:19]: Mapped values are capitalized by default?

File: context/app/static/js/components/Detail/FileBrowserDUA/utils.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
-function getDUAText(data_access_level) {
-  if (data_access_level === 'protected') {
+function getDUAText(mapped_data_access_level) {
+  if (mapped_data_access_level === 'protected') {
```
Comment (john-conroy) [inline context/app/static/js/components/Detail/FileBrowserDUA/utils.js:2]: If the values are now capitalized we need to change `protected` to `Protected`.

File: context/app/static/js/components/Detail/FileBrowserDUA/utils.js [hunk 3]
```diff
@@ -12,7 +12,7 @@ function getDUAText(data_access_level) {
     };
   }
 
-  if (data_access_level === 'consortium') {
+  if (mapped_data_access_level === 'consortium') {
```
Comment (john-conroy) [inline context/app/static/js/components/Detail/FileBrowserDUA/utils.js:15]: Also needs to be changed if values are capitalized.

**Comment by john-conroy** [comment]
Travis failing because of prettier. Should get to #973 this week.

File: context/app/static/js/components/Detail/FileBrowserDUA/utils.js (hunk 2 above)
Comment (mccalluc) [reply context/app/static/js/components/Detail/FileBrowserDUA/utils.js:2]: Sorry! Yes!

**Pull request merged**